zulipterminal/model.py

```diff
@@ -285,6 +286,13 @@ def react_to_message(self,
             response = self.client.add_reaction(reaction_to_toggle_spec)
         display_error_if_present(response, self.controller)
 
+    def session_draft_message(self) -> Optional[Message]:
+        return self.draft
+
+    def save_draft(self, message: Message) -> None:
+        self.draft = message
```

Comment:
I'm not sure why we're avoiding deepcopy here for encapsulation purposes.

Reply:
Fixed, also rename draft to _draft.